client/web/compose/src/components/PageBlocks/RecordListBase.vue

```diff
@@ -1792,6 +1852,10 @@ td:hover .inline-actions {
     color: $primary !important;
   }
 }
+
+::v-deep .b-form-tags-button {
```

Comment:
You should use this is the style non scoped section, or even better add it to custom.scss